translations/messages.fr.xlf

```diff
@@ -0,0 +1,3063 @@
+            &lt;li&gt;Que la première pièce de votre phalange atterrisse sur la première pièce rencontrée de la phalange à capturer.&lt;/li&gt;
+        &lt;/ol&gt;&lt;br/&gt;
+        Capturez la phalange.</target>
+      </trans-unit>
+      <trans-unit id="8213276201685541009" datatype="html">
+        <source>Congratulations, you made it!
+        Note that the diagonal phalanx was not aligned with your phalanx, hence even if it is longer, this does not prevent you from capturing some of its pieces on another alignment.</source>
+        <target>Bravo, vous avez réussi.
+        Constatez que la phalange diagonale n'étant pas alignée avec la notre, sa longueur supérieur n'empêche pas de capturer ses pièces dans un autre alignement.
+        </target>
+      </trans-unit>
+      <trans-unit id="4418812710815829575" datatype="html">
+        <source>Failed, you have not captured the phalanx.</source>
+        <target>Raté, vous n&apos;avez pas capturé la phalange.</target>
+      </trans-unit>
+      <trans-unit id="7226802484619632640" datatype="html">
+        <source>You can only capture when 4 or more of your pieces are aligned, and it is not the case.</source>
+        <target>Une capture ne peut que se faire si 4 pièces de votre couleurs sont alignées, ce n&apos;est pas le cas.</target>
+      </trans-unit>
+      <trans-unit id="6918785733984182442" datatype="html">
+        <source>You must select a valid capture that contains 4 pieces or more.</source>
+        <target>Veuillez choisir une capture valide qui contient 4 pièces ou plus.</target>
+      </trans-unit>
+      <trans-unit id="6602326768713192004" datatype="html">
+        <source>There are still possible captures to be done.</source>
+        <target>Il vous reste des captures à effectuer.</target>
+      </trans-unit>
+      <trans-unit id="2434818181880718873" datatype="html">
+        <source>Pieces must be inserted in spaces on the edge of the board.</source>
+        <target>Les pièces doivent être placée sur une case du bord du plateau.</target>
+      </trans-unit>
+      <trans-unit id="7875793227562861246" datatype="html">
+        <source>You must select a valid placement direction.</source>
+        <target>Veuillez choisir une direction valide pour le déplacement.</target>
+      </trans-unit>
+      <trans-unit id="3111415789460197695" datatype="html">
+        <source>You must select a placement alongside the insertion space.</source>
+        <target>Veuillez choisir un placement avec une direction.</target>
+      </trans-unit>
+      <trans-unit id="1848361274892061756" datatype="html">
+        <source>You cannot place a piece on a complete line.</source>
+        <target>Veuillez effectuer un placement sur une ligne non complète.</target>
+      </trans-unit>
+      <trans-unit id="2854410959652074326" datatype="html">
+        <source>This piece belongs to two captures. Please select another piece in the capture you&apos;d like to take.</source>
```

Comment:
you'd like to make.

Reply:
Ps: c'est pas plus sympa de rendre la case non cliquable que de lui permettre de faire un truc qui annule son mouvement et lui faire perdre son temps ?

Reply:
donc il clique dessus et il se passe rien ? Et il se demande pourquoi ? Ou bien toaster sans annuler le mouvement ?

Reply:
Toaster sans annuler me semble très bien aussi !